### PR TITLE
update doc CHANGES log for future beta release #1616

### PR DESCRIPTION
**Pull request by RogerHaase**

### CODE REVIEW

**Comment by UlrichB22** [comment]
A few small suggestions on capitalization or typos:

```
20c20
< * generate readthedocs content using python 3.10
---
> * generate readthedocs content using Python 3.10
29c29
< * add language auto-detect to user peraonal settings
---
> * add language auto-detect to user personal settings
33c33
< * Fix CI workflow to test with all supported python versions, run py39 with ubuntu 22.04
---
> * Fix CI workflow to test with all supported python versions, run py39 with Ubuntu 22.04
38c38
< * itemlinks beginning with + were incorrectly given non-existant class (+meta/Home)
---
> * itemlinks beginning with + were incorrectly given non-existent class (+meta/Home)
40c40
< * add validity check if user changes name/alias/display-name in UserSettings>Presonal
---
> * add validity check if user changes name/alias/display-name in UserSettings>Personal
```

File: docs/changes/CHANGES [hunk 1]
```diff
@@ -1,12 +1,47 @@
...
+Fixes
+~~~~~
+* added config option to enable or disable contenttypes
+* mass source code changes using pyupgrade to upgrade syntax to 3.9
+* removed unused imports
+* create SECURITY.md
+* generate readthedocs content using Python 3.10
+* remove use of obsolete setuptools
+* change use of whoosh lru cache to lfu cache
+* update Flask and Werkzeug to >3.0.0
```
Comment (ThomasWaldmann) [inline docs/changes/CHANGES:23]: Hmm, guess `>= 3.0.0`?

File: docs/changes/CHANGES [hunk 2]
```diff
@@ -1,12 +1,47 @@
...
+test carefully and report new issues and feature requests on the issue tracker:
+https://github.com/moinwiki/moin/issues.
+
+Fixes
+~~~~~
+* added config option to enable or disable contenttypes
+* mass source code changes using pyupgrade to upgrade syntax to 3.9
+* removed unused imports
+* create SECURITY.md
+* generate readthedocs content using Python 3.10
```
Comment (ThomasWaldmann) [inline docs/changes/CHANGES:20]: For my projects I now tend to only include stuff that is somehow "interesting" for users of the software or package maintainers and try to leave away minor details (one person is writing this, but potentially many people are reading this - saves them time).

File: docs/changes/CHANGES [hunk 3]
```diff
@@ -1,12 +1,47 @@
...
+* removed parentid to fix welcome/users_home_meta
+* mass source code changes after integrating "black" code auto-formater
+* supress false "unused import" when importing flatland.Form
+* replace deprecated datetime.utc* calls
+* add language auto-detect to user personal settings
+* readthedocs now has access to docs/examples
+* fix maint-validate-metadata to skip userprofiles backend
+* Add REV_NUMBER in sort for history view replacing mtime
+* Fix CI workflow to test with all supported python versions, run py39 with Ubuntu 22.04
+* add support for SQLAlchemy 2.0
```
Comment (ThomasWaldmann) [inline docs/changes/CHANGES:34]: `>= 2.0`?

File: docs/changes/CHANGES [hunk 4]
```diff
@@ -1,12 +1,47 @@
...
+* macros with user editing errors should not fill log with tracebacks
+* cleanup redundant icon directories
+* add itemlinks validator replacing wikilinks validator where needed
+* itemlinks beginning with + were incorrectly given non-existent class (+meta/Home)
+* remove trailing /> from void elements to comply with W3C standard
+* add validity check if user changes name/alias/display-name in UserSettings>Personal
+* add test to prove icons from CSS URLs is a subset of files in icon dir
+
+Version 2.0.0alpha 2024-03-27
```
Comment (ThomasWaldmann) [inline docs/changes/CHANGES:43]: The git tag is `2.0.0a1`.

File: docs/changes/CHANGES [hunk 5]
```diff
@@ -1,12 +1,47 @@
...
+MoinMoin 2 Version History
+==========================
 
 Please note: It is recommended that existing wiki's be upgraded to the
 latest moin 1.9.x release before converting to Moin 2. However, this may
 not be a necessary step as the 1.9.x file structure has not changed recently.
 
-Version 2.0.0alpha
-------------------
+Version 2.0.0beta 2024-??-??
```
Comment (ThomasWaldmann) [inline docs/changes/CHANGES:8]: Guess the git tag would be `2.0.0b1`?

File: docs/changes/CHANGES [hunk 6]
```diff
@@ -37,6 +72,9 @@ Missing features from 1.9.x
 * SVG editor
 * LDAP may have missing dependencies depending upon OS flavor
 * WikiDicts have limited function, does not use Babel translations
+* language support limited to English, Deutsch, Portuguese (Brasil), Russian
```
Comment (ThomasWaldmann) [inline docs/changes/CHANGES:75]: Maybe rather "German".

Guess whoever does not understand English won't read the changelog anyway.

**Review by ThomasWaldmann — COMMENTED** [review]
Some minor nitpicks I found.

Ping me if you need help with the release (can't promise to do testing / development though).

**Pull request merged**